code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -422,17 +422,17 @@
 	ph = 11
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
-/datum/reagent/toxin/chloralhydrate/on_mob_life(mob/living/carbon/M)
+/datum/reagent/toxin/chloralhydrate/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
 	switch(current_cycle)
 		if(1 to 10)
-			M.add_confusion(2)
-			M.drowsyness += 2
+			M.add_confusion(2 * REM * delta_time)
+			M.drowsyness += 2 * REM * delta_time
 		if(10 to 50)
-			M.Sleeping(40)
+			M.Sleeping(40 * REM * delta_time)
 			. = 1
 		if(51 to INFINITY)
-			M.Sleeping(40)
-			M.adjustToxLoss((current_cycle - 50)*REM, 0)
+			M.Sleeping(40 * REM * delta_time)
+			M.adjustToxLoss(1 * (current_cycle - 50) * REM * delta_time, 0)
```

Comment:
Is the 1 for readability? You don't do similar things in similar situations

Reply:
Pretty much.